SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -56,15 +65,41 @@ public FormSubmissionsUseCase(IDatabaseGateway databaseGateway, IMongoGateway mo
             return foundSubmission?.ToDomain().ToResponse();
         }
 
-        public List<CaseSubmissionResponse> ExecuteListBySubmissionStatus(SubmissionState state)
+        public IEnumerable<CaseSubmissionResponse>? ExecuteGetByQuery(QueryCaseSubmissionsRequest request, FilterDefinition<CaseSubmission>? filter2 = null)
```

Comment:
Is filter2 perhaps a left over from some earlier implementation? Looks like it's not used at the moment